src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/page/customfooterlibs.html

```diff
@@ -13,6 +13,7 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */-->
+#set( $expstart = "${" )
```

Comment:
we already have a variable for dollar symbol .. ${symbol_dollar}

Reply:
I needed one for "${" so instead of using two variables, I created one just like here:
https://github.com/adobe/aem-project-archetype/blob/44c8ee0e51b0021eebcb71dc9e57018020a9971b/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/commerce/logo/logo.html#L16

let me know if that's ok.

Reply:
I'm still confused why can;t we use    
`<sly data-sly-test="${symbol_dollar} {page.data && page.dataLayerClientlibIncluded}" data-sly-call="${symbol_dollar}{clientlib.js @ categories='core.forms.components.commons.v1.datalayer', async=true}"></sly>`

Reply:
removed introduced variable.